src/models/policy/index.js

```diff
@@ -97,3 +98,17 @@ export function isCustomPolicy(type) {
 export function isNamedPolicy(type) {
   return namedPolicies.find((policy) => policy.name === type) !== undefined;
 }
+
+export const deletePoliciesContainer = async (containerIri, fetch) => {
+  try {
+    await deleteContainer(containerIri, { fetch });
+  } catch (err) {
+    if (
+      !isHTTPError(err.message, 409) &&
+      !isHTTPError(err.message, 404) &&
+      !isHTTPError(err.message, 403)
```

Comment:
Do we only want to specifically catch these three, or should this catch any error responses?

Reply:
We want to ignore these 3 and throw and error if only if it isn't one of these.